bin/composer-dependency-analyser

```diff
@@ -82,6 +82,24 @@ if (isset($providedOptions['help'])) {
     exit;
 }
 
+$composerJsonPath = isset($providedOptions['composer-json'])
+    ? ($cwd . "/" . $providedOptions['composer-json'])
+    : ($cwd . "/composer.json");
+
+try {
+    $composerJson = new ComposerJson($composerJsonPath);
+} catch (OurRuntimeException $e) {
+    $exit($e->getMessage());
+}
+
+// load vendor that belongs to given composer.json
+$autoloadFile = dirname($composerJsonPath) . '/' . $composerJson->vendorDir . '/autoload.php';
```

Comment:
Ideally, you need to [resolve](https://github.com/nette/utils/pull/308) `vendorDir` relative to `dirname($composerJsonPath)`. 

Same issue is when resolving path relative to CWD.